src/utils/dev/debounce/debounce.ts

```diff
@@ -0,0 +1,10 @@
+export const debounce = (callback: () => void, timeout = 100) => {
+  let timer: NodeJS.Timeout | null;
+  return () => {
```

Comment:
it should accept arguments and pass them to `callback`

Reply:
done